src/pages/lib-v1/auth/advanced/q/platform/[platform].mdx

```diff
@@ -1,4 +1,8 @@
 export const meta = {
   title: `Advanced workflows`,
-  description: `Learn more about advanced workflows in the Amplify auth category. This includes subscribing to events, identity pool federation, auth-related Lambda triggers and working with AWS service objects.`,
+  description: `Learn more about advanced workflows in the Amplify auth category. This includes subscribing to events, identity pool federation, auth-related Lambda triggers and working with AWS service objects.`
 };
+
```

Comment:
same comment as on other pages. Let's avoid pages with just one fragment. Let's avoid fragments in general. Use inline filters instead.